### PR TITLE
Header

### LINKED ISSUE #5: Header, Navigation bar
Build Navigation menu

- [ ]  with links to live-site all content is imported
- [ ]  Build Mobile first
- [ ]  Tablet version
- [ ]  Desktop
- [ ]  includes the right hand popup navigation

Screenshot 2024-05-02 at 9 59 45 AM

Screenshot 2024-05-02 at 10 00 02 AM

### PR DESCRIPTION
**Pull request by dave-fink**
Added header block
Fix #5

Improvements:
- dropdown links clickable area is now full width for desktop view

Todo (separate issues created)
- add auto complete
- add chat functionality 

Test URLs:
- Before: https://main--hubblehomes-com--aemsites.hlx.live/
- After: https://5-header-nav--hubblehomes-com--aemsites.hlx.live/
- Before: https://main--hubblehomes-com--aemsites.hlx.live/drafts/dfink/blank-test-page
- After: https://5-header-nav--hubblehomes-com--aemsites.hlx.live/drafts/dfink/blank-test-page

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -1,275 +1,329 @@
...
+  background: var(--dark-blue);
+  box-shadow: 0 6px 12px rgb(0 0 0 / 17%);
+}
+nav>div>ul>li.on>ul { display: block }
+
+nav>div>ul>li>ul>li {
+  text-transform: none;
+  padding: 6px 0 0;
+  font-size: .85em;
+  line-height: 1.6em;
```
Comment (bhellema) [inline blocks/header/header.css:168]: 1.3em tightens it up like they have.

File: blocks/header/header.css [hunk 2]
```diff
@@ -1,275 +1,329 @@
...
+  margin-left: -14px;
+  padding: 4px 0 14px;
+  background: var(--dark-blue);
+  box-shadow: 0 6px 12px rgb(0 0 0 / 17%);
+}
+nav>div>ul>li.on>ul { display: block }
+
+nav>div>ul>li>ul>li {
+  text-transform: none;
+  padding: 6px 0 0;
```
Comment (bhellema) [inline blocks/header/header.css:166]: Maybe a top 2px tightens up the entire dialog. It seems like they have a love for squeezing everything together.

**Review by bhellema — COMMENTED** [review]
Styles look great.  There is a global style that has the a:any-link that's impacting some of the links in the nav giving the links a stronger look than what the original design has.  I wonder if we want to override that in the header's nav and set the font-weight to normal for the "sub areas".

File: blocks/header/header.css [hunk 3]
```diff
@@ -1,275 +1,329 @@
...
-  line-height: 1;
+#promo span {
+  display: block;
+  font-size: .72em;
 }
 
-header nav a:any-link {
-  color: currentcolor;
+a#promo span::after {
+  content: ' >'; /* TODO: perhaps use font-awesome - determin if it's necessary */
```
[the remark this answers is not in the document]
Comment (dave-fink) [reply blocks/header/header.css]: adjusted

File: blocks/header/header.css [hunk 4]
```diff
@@ -1,275 +1,329 @@
...
 
-header nav p {
-  margin: 0;
-  line-height: 1;
+#promo span {
+  display: block;
+  font-size: .72em;
 }
 
-header nav a:any-link {
```
[the remark this answers is not in the document]
Comment (dave-fink) [reply blocks/header/header.css:56]: adjusted

File: blocks/header/header.css (hunk 2 above)
Comment (dave-fink) [reply blocks/header/header.css]: adjusted

File: blocks/header/header.css (hunk 1 above)
Comment (dave-fink) [reply blocks/header/header.css:169]: In the spirit of usability I intentionally added a little extra space here.

**Review by bhellema — APPROVED** [review]
Great start to styling this project! lgtm.

**Review by varunmitra — APPROVED** [review]

**Pull request merged**